README.md

```diff
@@ -75,6 +76,36 @@ GET /manifests
 GET /manifests/<name>
 ````
 
+## Determine whether partial file uploads are supported
+````
+OPTIONS /files
+````
+This returns the following header and body:
+````
+Server: ....
+Allow: GET,HEAD,POST,OPTIONS,TRACE
+Content-Type: Application/octet-binary
+
+{
```

Comment:
This discussion I read before doing this:
http://zacstewart.com/2012/04/14/http-options-method.html

I'm not sure that anyone has really done a comprehensive thing yet.  We can update the spec when and if something becomes common.